packages/json/lib/commands/MSET.ts

```diff
@@ -0,0 +1,25 @@
+import { RedisJSON, transformRedisJsonArgument } from '.';
+
+export const FIRST_KEY_INDEX = 1;
+
+export interface KeyPathValue {
+    key: string;
+    path: string;
+    value: RedisJSON;
+}
+
+export function transformArguments(keyPathValues: Array<KeyPathValue>): Array<string> {
+    if (keyPathValues.length === 0) {
```

Comment:
There is no need to do it client side, I'm not into making the error cases faster... let the server handle it.